### PR TITLE
updated container

### PR DESCRIPTION
**Pull request by Mropat**
This PR adds/fixes
- Update containerfile
- Create action to publish container on dockerhub 

**How to test**:
- [x] Check that container is published correctly on dockerhub
- [ ] Launch container locally 
```
sudo docker pull clinicalgenomics/niptool-stage && \
sudo docker run clinicalgenomics/niptool-stage --expose 8000 -p 27017:27017 -i
```

**Expected test outcome**:
- [ ] Take a screenshot and attach or copy/paste the output.

**Review:**
- [ ] code approved by
- [ ] tests executed by
- [ ] "Merge and deploy" approved by
Thanks for filling in who performed the code review and the test!

This [version](https://semver.org/) is a:
- [x] **MINOR** - when you add functionality in a backwards compatible manner

### CODE REVIEW

File: NIPTool/commands/base.py [hunk 1]
```diff
@@ -41,9 +41,7 @@ def serve_command(reload: bool, api: str):
 
     This command will serve the user interface (external) as default
     """
-    app = "NIPTool.main:external_app"
-    if api == "internal":
-        app = "NIPTool.main:internal_app"
+    app = f"NIPTool.main:{api}_app"
```
Comment (moonso) [inline NIPTool/commands/base.py:44]: cool 🕶️

**Review by moonso — APPROVED** [review]
👍

File: NIPTool/commands/base.py (hunk 1 above)
Comment (Mropat) [reply NIPTool/commands/base.py:44]: I have a problem :rofl:

Comment (moonso) [reply NIPTool/commands/base.py:44]: what is that?

Comment (Mropat) [reply NIPTool/commands/base.py:44]: Correcting code that worked anyway

Comment (moonso) [reply NIPTool/commands/base.py:44]: haha, this is improvements 😄

**Pull request merged**